packages/status-react/src/system/avatar/avatar.tsx

```diff
@@ -13,10 +13,19 @@ interface Props {
   src?: string
   color?: string
   colorHash?: number[][]
+  initialsCount?: number
```

Comment:
notice:

Reply:
When defining `props` for system components it's always better be more restrictive about public API. 

`number` is probably too arbitrary, because we want to allow only 1 or 2 and use number union type.

Reply:
Also, `initialsLength` feels more descriptive, because `count`, I think, refers to some number rather then `string` length.